examples/bri-3/src/bri/workgroup/workgroups/api/dtos/request/createWorkgroup.dto.spec.ts

```diff
@@ -0,0 +1,188 @@
+
+    // Assert
+    expect(errors.length).toBe(1);
+    expect(errors[0].property).toEqual('name');
+    expect(errors[0].constraints.isNotEmpty).toContain(
+      'name should not be empty',
+    );
+  });
+
+  it('should return error in case administrators not provided.', async () => {
+    // Arrange
+    const dto = {
+      name: ['name'],
+      securityPolicy: 'sec',
+      privacyPolicy: 'priv',
+      participantIds: ['subject'],
+      worksteps: [],
+      workflows: [],
+    };
+    const createWorkgroupDto = plainToInstance(CreateWorkgroupDto, dto);
+
+    // Act
+    const errors = await validate(createWorkgroupDto);
+
+    // Assert
+    expect(errors.length).toBe(1);
+    expect(errors[0].property).toEqual('administratorIds');
+    expect(errors[0].constraints.isNotEmpty).toContain(
+      'administratorIds should not be empty',
+    );
+  });
+
+  it('should return error in case securityPolicy not provided.', async () => {
+    // Arrange
+    const dto = {
+      name: ['name'],
+      administratorIds: ['1'],
+      privacyPolicy: 'priv',
+      participantIds: ['subject'],
+      worksteps: [],
+      workflows: [],
+    };
+    const createWorkgroupDto = plainToInstance(CreateWorkgroupDto, dto);
+
+    // Act
+    const errors = await validate(createWorkgroupDto);
+
+    // Assert
+    expect(errors.length).toBe(1);
+    expect(errors[0].property).toEqual('securityPolicy');
+    expect(errors[0].constraints.isNotEmpty).toContain(
+      'securityPolicy should not be empty',
+    );
+  });
+
+  it('should return error in case privacyPolicy not provided.', async () => {
+    // Arrange
+    const dto = {
+      name: ['name'],
+      administratorIds: ['1'],
+      securityPolicy: 'sec',
+      participantIds: ['subject'],
+      worksteps: [],
+      workflows: [],
+    };
+    const createWorkgroupDto = plainToInstance(CreateWorkgroupDto, dto);
+
+    // Act
+    const errors = await validate(createWorkgroupDto);
+
+    // Assert
+    expect(errors.length).toBe(1);
+    expect(errors[0].property).toEqual('privacyPolicy');
+    expect(errors[0].constraints.isNotEmpty).toContain(
+      'privacyPolicy should not be empty',
+    );
+  });
+
+  it('should return error in case participantIds not provided.', async () => {
+    // Arrange
+    const dto = {
+      name: ['name'],
+      administratorIds: ['1'],
+      privacyPolicy: 'priv',
+      securityPolicy: 'sec',
+      worksteps: [],
+      workflows: [],
+    };
+    const createWorkgroupDto = plainToInstance(CreateWorkgroupDto, dto);
+
+    // Act
+    const errors = await validate(createWorkgroupDto);
+
+    // Assert
+    expect(errors.length).toBe(1);
+    expect(errors[0].property).toEqual('participantIds');
+    expect(errors[0].constraints.isNotEmpty).toContain(
+      'participantIds should not be empty',
+    );
+  });
+
+  it('should return error in case an empty administratorIds array is provided.', async () => {
```

Comment:
please move this one next to the one that is testing for administratorIds is not provided at all. Same goes for other tests that test the same property, ideally they should be next to one another.

Reply:
Rearranged